src/Umbraco.Web.UI.Client/src/views/common/overlays/querybuilder/querybuilder.controller.js

```diff
@@ -1,14 +1,20 @@
 (function () {
     "use strict";
 
-    function QueryBuilderOverlayController($scope, templateQueryResource) {
+    function QueryBuilderOverlayController($scope, $element, templateQueryResource, assetsService, angularHelper) {
```

Comment:
The three extra injected dependencies don't seem to be used in this file so best to be neat & tidy and remove them please @madsrasmussen